unity-client/Assets/Scripts/Tests/TestHelpers.cs

```diff
@@ -1098,47 +1099,47 @@ public static IEnumerator UnloadAllUnityScenes()
 
         public static SceneController InitializeSceneController(bool usesWebServer = false)
         {
-            var sceneController = UnityEngine.Object.FindObjectOfType<SceneController>();
+            var main = UnityEngine.Object.FindObjectOfType<Main>();
```

Comment:
Not a big fan of having this kind of things in tests. But I understand we are just "porting" the old `SceneController` behaviour into this new `Main` entry point. 

Let's not forget about this in the future and find a more elegant solution!

Reply:
Of course. This code is ancient.